arroyo-df/src/lib.rs

```diff
@@ -630,8 +615,35 @@ impl Debug for AggregateCalculation {
 
 #[derive(Debug)]
 struct DataFusionEdge {
```

Comment:
Should this include the _timestamp column?

Reply:
Yeah, right now it's implicit in the schema field called _timestamp, but probably better to be explicit